src/components/Atoms/Input/Input.js

```diff
@@ -92,6 +95,7 @@ const Input = React.forwardRef(
           aria-describedby={hasAria ? id : undefined}
           ref={ref}
           prefixLength={prefix.length}
+          required
```

Comment:
Whoops, didn't pass in the prop for some reason

Reply:
Also wonder if we want to see the browser validation in addition to our own?

Reply:
Browser validation was overriding the react-form-hook validation (hence using NoValidate on the form itself), but I feel it's better UX and accessibility to have it marked as required when it is.

Reply:
Ah, I wasn't familiar with this `noValidate` prop, thanks!

If `required` improves accessibility, definitely a case to use it, however, if browser validation indeed overrides react-hook-form validation, it would be an issue to add it as part of this PR - since this component is already being used in production by other forms which _don't_ have the `noValidate` prop

Reply:
Hmm yeah, that sounds like a point. Okay, I'll remove this for now and keep any 'Optional' labelling local to the MPs

Reply:
It's nice have the Optional label available on the general Input component - as long as it's optional itself rather than the default 

Once contentful gets updated with latest CL, I'll add the label to any fundraiser form inputs that need it

Reply:
Will revisit after lunch